core/silkworm/chain/config_test.cpp

```diff
@@ -20,7 +20,7 @@
 
 namespace silkworm {
 
-TEST_CASE("Config lookup") {
+TEST_CASE("Settings lookup") {
```

Comment:
IMHO "chain config" sounds better than "chain settings". Erigon also uses "chain config".

Reply:
Right ... I didn't notice that. I believe is a byproduct of a replacement.
Amended in 73d359e